vladyslav.taran/second/task2/tamagochi.rb

```diff
@@ -0,0 +1,100 @@
+class Pet
+ attr_accessor :name, :health, :eat, :play, :sleep
+ 
+def initialize (name, health, eat, play, sleep )
+    @name = name
+    @health = health  
+    @play = play
+    @sleep = sleep 
+    @eat = eat
+  end
```

Comment:
this comment still not fixed, if you have any problem to fix it, please reach out to me if you need some help